app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int):
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    people_list = [Person(singel_person["name"], singel_person["age"])
+                   for singel_person in people]
+
+    for singel_person in people:
```

Comment:
Typo in word `single`
Also It'd be better to use just `person` as a name for this variable, because word `single` doesn't tell us anything new about the variable. It's clear that only one person is stored in the variable `person`